src/test/java/Homework17.java

```diff
@@ -0,0 +1,80 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class Homework17 extends BaseTest{
+
+    @Test
+    public void addSongToPlaylist() throws InterruptedException {
+
+        String expectedSongAddedMessage = "Added 1 Song into \"Test Pro Playlist.\"";
+
+        navigateTopage();
+        provideEmail("dirzo@gmail.com");
+        providePassword("Te$ter1234");
+        clickSubmit();
+        Thread.sleep(2000);
+        searchASong("Way Of The Samurai");
+        clickViewAll();
+        selectFirstSong();
+        clickAddtoBtn();
+        selectPlaylist();
+        Assert.assertEquals(getAddtoListMsg(), expectedSongAddedMessage);
+    }
+
+    private String getAddtoListMsg() {
+        WebElement messageFlash = driver.findElement(By.cssSelector("div.success.show"));
+        return messageFlash.getText();
+    }
+
+    private void selectPlaylist() throws InterruptedException {
+        WebElement playlist = driver.findElement(By.xpath("//a[normalize-space()='Prepare for battle']"));
+        playlist.click();
+        Thread.sleep(2000);
+    }
+
+    private void clickAddtoBtn() throws InterruptedException {
+        WebElement addToBtn = driver.findElement(By.cssSelector("button[title='Add selected songs to…']"));
+        addToBtn.click();
+        Thread.sleep(2000);
+    }
+
+    private void selectFirstSong() {
+        WebElement firstUp = driver.findElement(By.xpath("//tr[@class='song-item selected']"));
```

Comment:
change your xpath to  =` //section[@id='songResultsWrapper']//tr[@class='song-item'][1]`